pages/index.en-US.mdx

```diff
@@ -2,14 +2,15 @@
 title: React Hooks for Data Fetching
 ---
 
-import Callout from 'nextra-theme-docs/callout'
+import { Callout } from 'nextra-theme-docs'
 import Features from 'components/features'
-import Bleed from 'nextra-theme-docs/bleed'
+import { Bleed } from 'nextra-theme-docs'
 
 {
 // wrapped with {} to mark it as javascript so mdx will not put it under a p tag
 }
-{<h1 className="text-center font-extrabold md:text-5xl mt-8">SWR</h1>}
+{<h1 className="text-center text-4xl tracking-tighter font-extrabold md:text-5xl mt-8">SWR</h1>}
```

Comment:
H1 default style was https://github.com/shuding/nextra/blob/8c87fc03121fe50fef9766983185ce295705b563/packages/nextra-theme-docs/src/styles.css#L394-L398. 
But got remove in nextra v2.